terraform/elasticbeanstalk.tf

```diff
@@ -0,0 +1,73 @@
+﻿//S3 bucket containing application version
+data "aws_s3_bucket" "gpg-application-version-storage" {
+  bucket = "gpg-application-version-storage"
+}
+
+// Archive file 
+data "aws_s3_object" "gpg-archive-zip" {
+  bucket = data.aws_s3_bucket.gpg-application-version-storage.id
+  key    = "publish.zip"
+}
+
+// Application
+resource "aws_elastic_beanstalk_application" "gpg-application" {
+  name        = join("-",["gpg-application", var-env])
```

Comment:
Is this supposed to be `var-env` or `var.env`?

Reply:
Definitely the latter!